homeassistant/components/modbus/manifest.json

```diff
@@ -1,10 +1,9 @@
 {
   "domain": "modbus",
   "name": "Modbus",
-  "codeowners": ["@janiversen"],
+  "codeowners": [],
   "documentation": "https://www.home-assistant.io/integrations/modbus",
   "iot_class": "local_polling",
   "loggers": ["pymodbus"],
-  "quality_scale": "platinum",
```

Comment:
This should be `silver` now, as `gold` is the first scale that requires a codeowner.

Reply:
You are probably right that it could be silver, but having the new scale in mind it is more correct to remove score, and leave the decision to future codeowners.

Reply:
That scale is not active at this moment in time.

Reply:
I know the score, believe me.

But I still see it better, to remove the score and leave it that decision to others, which I think is a perfectly valid choice !

Ps. I only change the score, because hassfest demands it.

Reply:
> and leave it that decision to others

Yes, that is what my review is about. Please change it to silver.

../Frenck

Reply:
Where is it documented that an integration can only be downgraded to to the first possibly fitting level ?

As current codeowner I cannot set a score which I am not comfortable with ! and I think that is a valid decision.

Once I am no longer codeowner, scores can be changed without my approval.

Reply:
Still I as codeowner have the freedom of downgrading as I believe correct!

Once the PR is merged, you or others can do what you want.

Reply:
> Still I as codeowner have the freedom of downgrading as I believe correct!

That is not what codeownership means.

Please adjust this PR according to code review.

Reply:
Adjusted in [11bc08b](https://github.com/home-assistant/core/pull/122010/commits/11bc08b9dec2a0cd39bf72c32d0ad30c60c267a5)